conjureup/ui/views/bootstrapwait.py

```diff
@@ -38,7 +38,7 @@ def redraw_kitt(self):
                                             'bootstrap.err')
         out = check_output("tail -n 10 {}".format(bootstrap_stderrpath),
                            shell=True)
-        self.output.set_text(out)
+        self.output.set_text(out.stderr.decode())
```

Comment:
huh. I don't see that in my non-snappy run, I wonder what's different.
check_output does return bytes, so we should decode it here, but I guess it gets translated in urwid somewhere along the line, and that must be different for snappy runs?

Still, this change should cause a new traceback, since bytes objects do not have a stderr attribute.